### ISSUE TITLE
DBButton: empty prop `variant=""` creates unexpected combination of variants filled and outlined

### ISSUE THREAD

**Issue by mfranzke** [db-ui/mono#2444]
### Which generators are impacted?

- [ ] All
- [ ] HTML
- [ ] React
- [ ] Angular
- [ ] Vue
- [ ] Web components
- [ ] Power Apps

### Reproduction case

empty prop `variant=""` creates unexpected combination of variants filled and outlined

### Expected Behaviour

_tbd_

### Browser version

None

**Comment by nmerget** [comment]
@bruno-sch is it only in vue or can we reproduce it [here](https://db-ui.github.io/mono/review/main/components/button/overview)?

**Comment by bruno-sch** [comment]
@nmerget yes, this bug can be reproduced in any framework or just by using html + css.

**Comment by nmerget** [comment]
> @nmerget yes, this bug can be reproduced in any framework or just by using html + css.

If I open https://db-ui.github.io/mono/review/main/components/button/overview and change it in the dom nothing happens

**Comment by bruno-sch** [comment]
Sorry, my mistake: you have to use data-variant="" to see the bug. But anyway, I've already fixed the bug and linked the PR...

<img width="385" alt="Bildschirmfoto 2024-04-15 um 17 14 07" src="https://github.com/db-ui/mono/assets/163294364/d75d33b9-0647-441a-8b24-36e15b4deedb">
